stdlib/2and3/msilib/__init__.pyi

```diff
@@ -0,0 +1,133 @@
+
+import sys
+from typing import List, Tuple, Union, Set, Optional, Dict, Container, Any
+from types import ModuleType
+
+if sys.version_info >= (3, 8):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
+
+if sys.platform == 'win32':
+    from _msi import _Database
+
+    AMD64: bool
+    Itanium: bool
+    Win64: bool
+
+    datasizemask: Literal[0x00ff]
+    type_valid: Literal[0x0100]
+    type_localizable: Literal[0x0200]
+    typemask: Literal[0x0c00]
+    type_long: Literal[0x0000]
+    type_short: Literal[0x0400]
+    type_string: Literal[0x0c00]
+    type_binary: Literal[0x0800]
+    type_nullable: Literal[0x1000]
+    type_key: Literal[0x2000]
+    knownbits: Literal[0x3fff]
+
+    class Table:
+
+        name: str
+        fields: List[Tuple[int, str, int]]
+
+        def __init__(self, name: str) -> None: ...
+        def add_field(self, index: int, name: str, type: int) -> None: ...
+        def sql(self) -> str: ...
+        def create(self, db: _Database) -> None: ...
+
+    class _Unspecified: ...
+
+    def change_sequence(seq: List[Tuple[str, Optional[str], int]], action: str, seqno: Union[int, _Unspecified] = ..., cond: Union[str, _Unspecified] = ...) -> None: ...
```

Comment:
`Type[_Unspecified]` * 2 (needs import)

Also, `seq` can be any `Sequence`. (Also needs import.)